dxr/plugins/core.py

```diff
@@ -429,6 +429,11 @@ class FileToIndex(dxr.indexers.FileToIndex):
     def __init__(self, path, contents, plugin_name, tree, vcs):
         super(FileToIndex, self).__init__(path, contents, plugin_name, tree)
         self.vcs = vcs
+        self.extension = splitext(self.path)[1].lower()
```

Comment:
https://en.wikipedia.org/wiki/Scalable_Vector_Graphics suggests that "SVG" is not an acceptable extension, but maybe there's a more official spec somewhere else.